Demo/Demo/SwiftUIComponents/PayPalWebViews/CreateOrderPayPalWebView.swift

```diff
@@ -5,6 +5,8 @@ struct CreateOrderPayPalWebView: View {
     @ObservedObject var paypalWebViewModel: PayPalWebViewModel
 
     @State private var selectedIntent: Intent = .authorize
+    @State private var vaultCustomerID: String = ""
```

Comment:
I thought you had said during stand that regardless of what's customer ID is passed in the API always generates one? If that's the case, I think we should remove this from our demo.

Reply:
Yes, my plan is to do that today.

Reply:
Addressed in  commit dc0c32e4f431174aa5db43e4a7155b266a97f774